x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/user/GetUserPrivilegesResponseTests.java

```diff
@@ -38,12 +41,13 @@
 import java.util.stream.Collectors;
 
 import static java.util.Collections.emptySet;
+import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 
 public class GetUserPrivilegesResponseTests extends ESTestCase {
```

Comment:
Will roles be allowed to mix "indices" and "remote_indices"? If yes or no, it might be useful to add a test for that scenario to ensure whatever we chose is working as designed. If someone comes along later and toggles it, a unit test would catch that issue.

Reply:
Hmm I'm not positive about what you mean by mix `indices` and `remote_indices`, could you clarify?

The `Role` class has two separate fields, `indices` and `remoteIndices` so they are separate here and cannot be mixed.

Within the test, when we generate a response, the response is randomized and can have 0 or more indices and 0 or more remote indices, so we are covering cases where the response contains both indices and remote indices.

Reply:
I think I am conflating `QC user role descriptor(s)` vs `FC user role descriptor`. I think it is safe for QC roles to include indices and remote_indices. I am going to open a new issue for `FC user role descriptor`.

Reply:
Resolving this thread. Having indices and remote_indices in same role is OK.